packages/core/src/sdk/product/useProduct.ts

```diff
@@ -2,23 +2,23 @@ import { gql } from '@faststore/graphql-utils'
 import { useMemo } from 'react'
 
 import type {
-  BrowserProductQueryQuery,
-  BrowserProductQueryQueryVariables,
+  ClientProductQueryQuery,
+  ClientProductQueryQueryVariables,
 } from '@generated/graphql'
 
 import { useQuery } from '../graphql/useQuery'
 import { useSession } from '../session'
 
 const query = gql`
-  query BrowserProductQuery($locator: [IStoreSelectedFacet!]!) {
```

Comment:
That's because `@faststore/graphql-utils` [understand operations](https://github.com/vtex/faststore/blob/bddf04308826369f86322851b554cc58ab5b2161/packages/graphql-utils/src/index.ts#L33) ends with `Query`, and then codegen creates persisted `ClientProductQuery` and `ClientProductQueryQuery` types.

Reply:
Maybe this is something that we can revisit in the future. After all, we are the authors of graphql-utils, right?

Reply:
right! This is used to differentiate POST and GET methods. If we have another way to diff it, we could change this logic.